nengo_dl/config.py

```diff
@@ -22,7 +22,7 @@ def configure_settings(**kwargs):
     Parameters
     ----------
     trainable : bool or None
-        Adds a parameter to Nengo Ensembles/Connections/Networks that controls
+        Adds a parameter to Nengo Ensembles/Connections that controls
```

Comment:
Is this a change in function that this can't be set on networks, or just a fix to the documentation?

Reply:
Just a fix for the documentation (setting it on networks was removed a while ago, I just missed this docstring).